src/AltinnCore/RuntimeDataService/Repository/IReporteeElementRepository.cs

```diff
@@ -0,0 +1,31 @@
+using System.Threading.Tasks;
+using AltinnCore.Runtime.DataService.Models;
+
+namespace AltinnCore.Runtime.DataService.Repository
+{
+    public interface IReporteeElementRepository
+    {
+        /// <summary>
+        /// Get reportee element details for the given parameters
+        /// </summary>
+        /// <param name="reporteeId">the owner of the reportee element</param>
+        /// <param name="reporteeElementId">the reportee element id</param>
+        /// <returns>The form data for the given parameters</returns>
+        Task<ReporteeElement> GetReporteeElementFromCollectionAsync(string reporteeId, string reporteeElementId);
```

Comment:
Is there a reason that instanceId not beeing a guid? (since it will be that?)
Is there a reason reporteeId is not a int?

Reply:
Fixed